_posts/2022-08-03-using-h3-hexes.md

```diff
@@ -0,0 +1,126 @@
+
+## H3: The Solution
+{: .text-left}
+
+Uber created and open-sourced the [H3 geospatial indexing system](https://h3geo.org/) to solve these problems. The grid used by H3 is a mathematically-defined series of hexagons arranged in tiers or resolutions such that each hexagon _logically_ completely contains seven child hexagons that are _geographically_ mostly contained by the larger hexagon.
+
+![H3 Hexagons at various scales]({% link images/h3_hexes.png %})
+{: .flex .flex--center}
+
+### The Basics
+{: .text-left}
+
+To understand how H3 solves these problems, we need to understand how it works. Coming from the GIS world, we think of the "point in polygon" problem spatially: search to see if the given point is inside the given area. To understand H3, we need to shift our thinking. **H3 takes a lat/long and assigns an ID that corresponds to a mathematically-defined hex.** There's nothing inherently spatial about it, it's all just math. The output is just a number.
+
+### Common Grids
+{: .text-left}
+
+As an open-source system, H3 has become a sort of de facto standard. Anyone can use it, and anyone can take someone else's data with H3 grid ids and compare it with their own. While it's been around for a few years, there's a definite "flavor of the month" aspect at play so it remains to be seen how widely it will be adopted.
+
+However, even if it doesn't become the overwhelming global standard, the H3 system can still be useful for **standardizing analyses across projects within an organization.** If we at UGRC do a project for one state agency with the grid, it would be easy to pull that data into another project down the road.
+
+### Scaling Grid Resolution
+{: .text-left}
+
+It is mathematically impossible<sup>1</sup> to create a larger hexagon from smaller hexagons, which makes creating a perfect _geographical_ hierarchy of hexes impossible. However, H3 makes a _logical_ hierarchy by rotating the larger, parent hex to cover as much of its seven children as possible, as visible in the image above. The H3 [indexing page](https://h3geo.org/docs/highlights/indexing) has more detail on how this works.
+
+Using these multiple scales (which H3 calls resolutions), we can **create hexes at multiple scales to aggregate our data to larger and larger areas.** With these, we can view trends at the neighborhood, city, or county level. We can also create web maps that allow us to drill down from general trends to neighborhood or block level to see what areas contribute the most.
+
+### Performance
+{: .text-left}
+
+As noted above, H3 use some fancy mathematics under the hood to assign hex ids instead of doing spatial analysis. Written in C and exposed through a variety of bindings in other languages, these calculations are **pretty darn fast at figuring out what hex cell a point belongs in.**
```

Comment:
**chore**: grammar? 
```suggestion
As noted above, H3 uses some fancy mathematics under the hood to assign hex ids instead of doing spatial analysis. Written in C and exposed through a variety of bindings in other languages, these calculations are **pretty darn fast at figuring out what hex cell a point belongs in.**
```